cmd/augerctl/main.go

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Suggest adding a Markdown file at this level outlining the intent for `augerctl` and basic usage. Can re-use example for this pr description.